service/src/main/java/bio/terra/cbas/dao/RunDao.java

```diff
@@ -29,14 +31,19 @@ public int createRun(Run run) {
         new BeanPropertySqlParameterSource(run));
   }
 
-  public List<Run> retrieve() {
+  public List<Run> getRuns() {
```

Comment:
This was a personal preference choice... happy to revert if people don't like it!

Reply:
I like it